CRM/MembershipExtras/Job/OverdueMembershipPeriodProcessor.php

```diff
@@ -13,6 +13,12 @@ class CRM_MembershipExtras_Job_OverdueMembershipPeriodProcessor {
    * @throws \Exception
    */
   public function run() {
+    $autoUpdateOverdueMembershipEnabled = CRM_MembershipExtras_SettingsManager::shouldUpdateOverdueMembership();
+    if (!$autoUpdateOverdueMembershipEnabled) {
+      return TRUE;
+    }
+
+    $autoUpdateOverdueMembershipEnabled = CRM_MembershipExtras_SettingsManager::shouldUpdateOverdueMembership();
```

Comment:
This line can be deleted, right? I mean, it's exactly the same as ln 16, $autoUpdateOverdueMembershipEnabled should still have the same value.